docs/curate.md

```diff
@@ -5,4 +5,5 @@
 
 curate-df
 curate-any
+curate-subclass
```

Comment:
If this is the recommended way to curate custom datasets we should put it first. Or we integrate it into curate-any and the curate-any guide starts with subclassing the Curator and only then walks through how to do it low-level.

3 documents on curation is potentially a bit intimidating in particular when curate-any will almost never be used and should be more like a footnote?

Reply:
I think it is the recommended way if you want to enforce structure for a specific format and especially across people. However, if you're curating a dataset as a one off, subclassing `Curator` is too complex and annoying. Then the content in `curate-any` applies better I'd argue.